tasks/Functions-admin/setOcrConfig.js

```diff
@@ -7,38 +7,28 @@ task("functions-set-ocr-config", "Sets the OCR config using values from a given
       throw Error("This command cannot be used on a local development chain.  Specify a valid network.")
     }
 
-    let overrides = undefined
-    if (network.name === "goerli") {
-      overrides = {
-        // be careful, this may drain your balance quickly
-        maxPriorityFeePerGas: ethers.utils.parseUnits("50", "gwei"),
-        maxFeePerGas: ethers.utils.parseUnits("50", "gwei"),
-      }
+    let ocrConfig
+    try {
+      ocrConfig = require("../../FunctionsOracleConfig.json")
+    } catch (error) {
+      console.log(
+        'No Oracle configuration file found. Generate and add "FunctionsOracleConfig.json" to the root of this repository.'
+      )
+      throw new Error(error)
     }
 
-    const oracleFactory = await ethers.getContractFactory("FunctionsOracle")
-    const oracle = oracleFactory.attach(networkConfig[network.name]["functionsOracle"])
+    const oracleFactory = await ethers.getContractFactory("contracts/dev/functions/FunctionsOracle.sol:FunctionsOracle")
+    const oracle = oracleFactory.attach(networkConfig[network.name]["functionsOracleProxy"])
 
-    const ocrConfig = require("../../" + taskArgs.configfile)
```

Comment:
Heads up - we don't want to delete that.

Reply:
Oops, bad merge conflict resolution. Good catch.